### ISSUE TITLE
Decisions on species filtering

### ISSUE THREAD

**Issue by Lewis-Barnett-NOAA** [DFO-NOAA-Pacific/surveyjoin#40]
I have been tinkering with the AFSC data to see how to filter species by data richness to keep the catch data to a manageable size for download.  I will share some plots to help decide on the inclusion criteria, but we may need to first make some decisions about how to handle invertebrate taxa.

Filtering by frequency of occurrence and total catch among all AK surveys narrows the species set down quite a bit (see list of species below with > 0.05 frequency of occurrence and > 25000kg total weight). However, there are still many invertebrates in the list that we may want to exclude due to vastly different catchabilities among surveys with different footropes.  Filtering out organisms identified at higher taxonomic levels than the species level does reduce the number of ambiguous invertebrate taxa, but also leads to dropping some groups of interest (e.g., porifera).

![image](https://github.com/DFO-NOAA-Pacific/surveyjoin/assets/1361087/dfc326b0-222e-4833-8ff6-9258abd46d42)

**Comment by Lewis-Barnett-NOAA** [comment]
If we can't find a satisfying general approach to the invertebrates, one thing that may help in this case is filtering by frequency of occurrence within each individual survey in Alaska, as that would help downweight all the benthic inverts captured by the Bering shelf surveys.

**Comment by ecophilina** [comment]
Could there be a few different data batches like: everything over a certain occurrence threshold, vertebrates under said threshold, and invertebrates under said threshold?

**Comment by Lewis-Barnett-NOAA** [comment]
Yes, following the idea of having a high and low scope species set, I'll aim to create these separately for fish and inverts.  So their would be 4 catch data versions, where the user can combine fish and inverts as desired.

**Comment by Lewis-Barnett-NOAA** [comment]
After experimenting a bunch with different thresholds, I created 6 subsets of species (high and low scope for each of 3 taxonomic groups: fishes, structure-forming invertebrates (corals and sponges only), and other invertebrates).  See commit 18ba129 for the different thresholds by group.  

I found that these thresholds gave a relatively balanced number of species, ranging from around 100 fishes or invertebrates combined at the high end and ~30 at the low end, and included most of the species I suspected would be of general interest.  Happy to refine these based on the group feedback and how much of the list intersects with the data from other regions.
